### PR TITLE
Update main.tf

### PR DESCRIPTION
**Pull request by himanso**
the command was unable to find the cluster without the region parameter. With region parameter it is working fine

aws eks update-kubeconfig --name ${var.cluster_name} --region ${var.region}

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: modules/cluster/main.tf [hunk 1]
```diff
@@ -117,7 +117,7 @@ resource "null_resource" "kubeconfig" {
     module.eks
   ]
   provisioner "local-exec" {
-    command     = "aws eks update-kubeconfig --name ${var.cluster_name}"
+    command     = "aws eks update-kubeconfig --region us-east-1 --name ${var.cluster_name}"
```
Comment (ankitm123) [inline modules/cluster/main.tf:120]: I dont think this is the root cause of the issue. What happens if the eks cluster is in ca-central-1 or any region other than us-east-1.
I think there are 2 ways to fix this:
* Explicitely specify a region in the provider block in the main.tf:
```
provider "aws" {
  region  = "us-east-1"
}
```
* Specify region as an environment variable:
```
AWS_REGION=us-east-1 terraform apply
```

Comment (patrickleet) [reply modules/cluster/main.tf:120]: And if you are using AWS_PROFILE, make sure there is a matching `config` file with the region for that profile set

Comment (arpad9) [reply modules/cluster/main.tf:120]: I also addressed this in #102.

**Comment by ankitm123** [comment]
Going to close this PR @himanso as #102 fixes the doc issue.

**Pull request closed without merging**